src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -541,10 +572,63 @@ public Visitable visit(MethodCallExpr method, Void p) {
     return super.visit(method, p);
   }
 
+  @Override
+  public Visitable visit(ClassOrInterfaceType typeExpr, Void p) {
+    /**
+     * Workaround for a JavaParser bug: When a type is referenced as a class path, like
+     * com.example.Dog dog, JavaParser considers its package components (com and com.example) as
+     * types, too. This issue happens even when the source file of the Dog class is present in the
+     * codebase.
+     */
+    if (!isCapital(typeExpr.getName().asString())) {
```

Comment:
This is going to cause us problems if there is a user-written class that begins with a non-capital, I think. That's allowed in Java (but rare, because convention says class names should begin with capitals). However, if we're going to make this assumption about the input, we need to enforce it: that is, Specimin needs to crash if one of the input files contains a class with a lowercase name. Please add a test case that demonstrates that crash, once you've added it.

Reply:
3c414c21216c89a421d4f039cb6642849b47d27b